mm2src/coins/utxo/utxo_builder/utxo_arc_builder.rs

```diff
@@ -135,6 +177,28 @@ async fn merge_utxo_loop<T>(
 ) where
     T: UtxoCommonOps + GetUtxoListOps,
 {
+    let (my_address, script_pubkey) = match weak.upgrade() {
+        Some(arc) => {
+            let coin = constructor(arc);
+            let my_address = match coin.as_ref().derivation_method {
+                DerivationMethod::SingleAddress(ref my_address) => my_address,
+                DerivationMethod::HDWallet(_) => {
+                    warn!("'merge_utxo_loop' is currently not used for HD wallets");
+                    return;
+                },
+            };
+            let script_pubkey = match output_script(my_address) {
+                Ok(script) => script,
+                Err(e) => {
+                    error!("Error {} on output_script for coin {}", e, coin.as_ref().conf.ticker);
```

Comment:
Similar to `covered_warn`, you can create `covered_error` macro and use it here.

Reply:
[df83321](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2587/commits/df83321ef778cd40b82bee171aa4ece14000c7e7)